EIPS/eip-1967.md

```diff
@@ -42,6 +42,12 @@ Storage slot `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
 
 Holds the address of the logic contract that this proxy delegates to.
 
+### Beacon contract address
+
+Storage slot `0xa3f0ad74e5423aebfd80d3ef4346578335a9a72aeaee59ff6cb3582b35133d50` (obtained as `bytes32(uint256(keccak256('eip1967.proxy.beacon')) - 1)`).
+
+Holds the address of the beacon contract this proxy relies on. This is an alternative to the logic contract slot. See the **Beacons** section below.
```

Comment:
I think we have to add something like "optional" here, and/or to specify that if both beacon and implementation are present which one should take precedence.

Reply:
This is what I tried to mean when I say its an alternative to the logic slot. I'll try to rephrase that/